src/ddmd/templateparamsem.d

```diff
@@ -1,5 +1,19 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Template implementation.
+ *
+ * Copyright:   Copyright (c) 1999-2017 by Digital Mars, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/ddmd/templateparamsem.d, _templateparamsem.d)
+ */
+
 module ddmd.templateparamsem;
 
+// Online documentation: https://dlang.org/phobos/ddmd_templateparamsem.html
```

Comment:
That URL results in a 404

Reply:
The link is correct, the file is forthcoming.